app/experimenter/experiments/migrations/0163_auto_20210325_2041.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 3.1.7 on 2021-03-25 20:41
+
+from django.db import migrations, models
+
+from experimenter.experiments.constants import NimbusConstants
+
+
+def update_changelogs(apps, schema_editor):
+    NimbusChangeLog = apps.get_model("experiments", "NimbusChangeLog")
+    db_alias = schema_editor.connection.alias
+    NimbusChangeLog.objects.using(db_alias).filter(old_status="REVIEW").update(
+        old_status=NimbusConstants.Status.DRAFT.name,
+        old_publish_status=NimbusConstants.PublishStatus.REVIEW.name,
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(old_status="ACCEPTED").update(
+        old_status=NimbusConstants.Status.DRAFT.name,
+        old_publish_status=NimbusConstants.PublishStatus.WAITING.name,
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(new_status="REVIEW").update(
+        new_status=NimbusConstants.Status.DRAFT.name,
+        new_publish_status=NimbusConstants.PublishStatus.WAITING.name,
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(new_status="ACCEPTED").update(
+        new_status=NimbusConstants.Status.DRAFT.name,
+        new_publish_status=NimbusConstants.PublishStatus.WAITING.name,
+    )
```

Comment:
The values here are still incorrect.  The database values are `"Review"` and `"Accepted"`.  You can see in the constants above, the string literals are the values that are stored in the db:

```py
        REVIEW = "Review"
        ACCEPTED = "Accepted"
```